iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -164,428 +220,55 @@ TYPED_TEST(ServiceDiscovery_test, ReofferedServiceWithServiceDescriptionCanBeFou
     EXPECT_THAT(*serviceContainer.begin(), Eq(SERVICE_DESCRIPTION));
 
     producer.stopOffer();
-    this->InterOpWait();
+    this->waitUntilEventuallyNotFound(SERVICE_DESCRIPTION);
+
     this->findService(SERVICE_DESCRIPTION.getServiceIDString(),
                       SERVICE_DESCRIPTION.getInstanceIDString(),
                       SERVICE_DESCRIPTION.getEventIDString());
```

Comment:
```suggestion
    this->findService(SERVICE_DESCRIPTION);
```
This can be replaced in the whole test and probably also in other tests.

Reply:
On the other hand, this can be even removed here since `waitUntilEventuallyNotFound` and `waitUntilEventuallyFound` already do a `findService`.

Reply:
Yes they do. I will remove the redundancy.